ios/FluentUI.Demo/FluentUI.Demo/Demos/SearchBarDemoController.swift

```diff
@@ -7,6 +7,23 @@ import FluentUI
 import UIKit
 
 class SearchBarDemoController: DemoController, SearchBarDelegate {
+    private lazy var searchBarWithBadgeView: SearchBar = {
```

Comment:
nit: you can do that with a bit less code:

   ```
 private lazy var searchBarWithBadgeView: SearchBar = {
        buildSearchBar(autocorrectionType: .yes,
                       placeholderText: "Type badge to add a badge")
    }()
```

Reply:
Done

Reply:
nit: do you mind adjusting the indentation so that the buildSearchBar call uses only 2 lines? (it's the pattern we've been using throughout the library)
```
        buildSearchBar(autocorrectionType: .yes,
                       placeholderText: "Type badge to add a badge")
```